volcano scheduler panic  with  resourcequota plugin enabled when no resourcequota in a namespace

<!-- This form is for bug reports ONLY!

If you're looking for a help then check our [Slack Channel](https://cloud-native.slack.com/messages/volcano) or have a look at our [dev mailing](https://groups.google.com/forum/#!forum/volcano-sh) 
-->

**What happened**:
volcano scheduler panic  with  resourcequota plugin enabled when no resourcequota in  namespace 

**What you expected to happen**:
volcano-scheduler doesnot crash
**How to reproduce it (as minimally and precisely as possible)**:
1. set the resourcequota plugin in scheduler config

```
  volcano-scheduler.conf: |
    actions: "enqueue, allocate, backfill, preempt"
    tiers:
    - plugins:
      - name: priority
      - name: resourcequota
      - name: conformance
      - name: overcommit
        arguments:
          overcommit-factor: 10.0
    - plugins:
      - name: drf
      - name: gang
      - name: predicates
      - name: proportion
      - name: nodegroup
        enablePredicate: true
        enableNodeOrder: true
      - name: nodeorder
      - name: binpack
```
2. `test` namespace has no resourcequota

```
[root@host-10-19-37-28 volcano]# kubectl -n test get resourcequota
No resources found in test namespace.
```
3. define a queue and a job 
```
[root@host-10-19-37-28 volcano]# cat queuetest.yaml
apiVersion: scheduling.volcano.sh/v1beta1
kind: Queue
metadata:
  name: test
spec:
  weight: 1
  reclaimable: false
  capability:
    cpu: 6
  affinity:
    nodeGroupAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        - gzf
      preferredDuringSchedulingIgnoredDuringExecution:
        - noprefer
  guarantee:
    resource:
      cpu: 1

[root@host-10-19-37-28 volcano]# cat queuejob.yaml
apiVersion: batch.volcano.sh/v1alpha1
kind: Job
metadata:
  name: vc-job4
  namespace: test
spec:
  # minAvailable: 0
  schedulerName: volcano
  queue: test
  priorityClassName: high-priority
  policies:
    - event: PodEvicted
      action: RestartJob
  tasks:
    - replicas: 1
      name: job4
      policies:
      - event: TaskCompleted
        action: CompleteJob
      template:
        spec:
          priorityClassName: high-priority
          containers:
            - command:
              - sleep
              - 10m
              image: nginx:latest
              name: nginx
              resources:
                requests:
                  cpu: 2
                limits:
                  cpu: 2
          restartPolicy: OnFailure
```
4. create job, volcano scheduler panic
```
[root@host-10-19-37-28 volcano]# kubectl create -f ./queuejob.yaml
job.batch.volcano.sh/vc-job4 created
[root@host-10-19-37-28 volcano]# kubectl -n volcano-system get po
NAME                                  READY   STATUS             RESTARTS     AGE
volcano-admission-5dcb9c44c5-bk759    1/1     Running            0            7d6h
volcano-admission-init--1-zbtvc       0/1     Completed          0            7d6h
volcano-controllers-f4b69577b-ntvq5   1/1     Running            0            7d6h
volcano-scheduler-c98cb745b-vgbq4     0/1     CrashLoopBackOff   1 (6s ago)   105s
```
5. check the volcano-scheduler log
```
I0905 09:55:04.419942       1 session.go:202] Close Session bf0a2cf1-41b7-4cd2-98bb-ee8921ee8c33
E0905 09:55:04.420304       1 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)
goroutine 286 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic({0x1991240?, 0x2cdaec0})
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:74 +0x86
k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0xfffffffe?})
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:48 +0x75
panic({0x1991240, 0x2cdaec0})
	/usr/local/go/src/runtime/panic.go:838 +0x207
volcano.sh/volcano/pkg/scheduler/plugins/resourcequota.(*resourceQuotaPlugin).OnSessionOpen.func1({0x1bbc3c0?, 0xc00079a870})
	/go/src/volcano.sh/volcano/pkg/scheduler/plugins/resourcequota/resourcequota.go:48 +0xaa
volcano.sh/volcano/pkg/scheduler/framework.(*Session).JobEnqueueable(0xc000bc1340, {0x1bbc3c0, 0xc00079a870})
	/go/src/volcano.sh/volcano/pkg/scheduler/framework/session_plugins.go:395 +0x1bf
volcano.sh/volcano/pkg/scheduler/actions/enqueue.(*Action).Execute(0x1edf090?, 0xc000bc1340)
	/go/src/volcano.sh/volcano/pkg/scheduler/actions/enqueue/enqueue.go:94 +0x9bb
volcano.sh/volcano/pkg/scheduler.(*Scheduler).runOnce(0xc0001a8a00)
	/go/src/volcano.sh/volcano/pkg/scheduler/scheduler.go:116 +0x348
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc0006e5f00?)
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:155 +0x3e
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0x1edd5c0?, {0x1eba040, 0xc000517f80}, 0x1, 0xc00051e000)
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:156 +0xb6
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00051e4e0?, 0x3b9aca00, 0x0, 0x10?, 0xc0005a1710?)
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133 +0x89
k8s.io/apimachinery/pkg/util/wait.Until(0xc0005a17d0?, 0xa23d86?, 0xc0007420c0?)
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:90 +0x25
created by volcano.sh/volcano/pkg/scheduler.(*Scheduler).Run
	/go/src/volcano.sh/volcano/pkg/scheduler/scheduler.go:91 +0x1aa
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x174a88a]

goroutine 286 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0xfffffffe?})
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:55 +0xd8
panic({0x1991240, 0x2cdaec0})
	/usr/local/go/src/runtime/panic.go:838 +0x207
volcano.sh/volcano/pkg/scheduler/plugins/resourcequota.(*resourceQuotaPlugin).OnSessionOpen.func1({0x1bbc3c0?, 0xc00079a870})
	/go/src/volcano.sh/volcano/pkg/scheduler/plugins/resourcequota/resourcequota.go:48 +0xaa
volcano.sh/volcano/pkg/scheduler/framework.(*Session).JobEnqueueable(0xc000bc1340, {0x1bbc3c0, 0xc00079a870})
	/go/src/volcano.sh/volcano/pkg/scheduler/framework/session_plugins.go:395 +0x1bf
volcano.sh/volcano/pkg/scheduler/actions/enqueue.(*Action).Execute(0x1edf090?, 0xc000bc1340)
	/go/src/volcano.sh/volcano/pkg/scheduler/actions/enqueue/enqueue.go:94 +0x9bb
volcano.sh/volcano/pkg/scheduler.(*Scheduler).runOnce(0xc0001a8a00)
	/go/src/volcano.sh/volcano/pkg/scheduler/scheduler.go:116 +0x348
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc0006e5f00?)
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:155 +0x3e
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0x1edd5c0?, {0x1eba040, 0xc000517f80}, 0x1, 0xc00051e000)
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:156 +0xb6
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00051e4e0?, 0x3b9aca00, 0x0, 0x10?, 0xc0005a1710?)
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133 +0x89
k8s.io/apimachinery/pkg/util/wait.Until(0xc0005a17d0?, 0xa23d86?, 0xc0007420c0?)
	/go/src/volcano.sh/volcano/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:90 +0x25
created by volcano.sh/volcano/pkg/scheduler.(*Scheduler).Run
	/go/src/volcano.sh/volcano/pkg/scheduler/scheduler.go:91 +0x1aa
```

**Anything else we need to know?**:

**Environment**:
- Volcano Version:  install with `kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml`
- Kubernetes version (use `kubectl version`): 1.22
- Cloud provider or hardware configuration:  N/A
- OS (e.g. from /etc/os-release):  centos 7
- Kernel (e.g. `uname -a`): `3.10.0-957.27.2.el7.x86_64 #1 SMP Mon Jul 29 17:46:05 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`
- Install tools: N/A
- Others:

--- a/pkg/scheduler/plugins/resourcequota/resourcequota.go
+++ b/pkg/scheduler/plugins/resourcequota/resourcequota.go
@@ -45,6 +45,10 @@ func (rq *resourceQuotaPlugin) OnSessionOpen(ssn *framework.Session) {
 			return util.Permit
 		}
 
+		if ssn.NamespaceInfo[api.NamespaceName(job.Namespace)] == nil {
+			return util.Permit
+		}
+
 		quotas := ssn.NamespaceInfo[api.NamespaceName(job.Namespace)].QuotaStatus
 		for _, resourceQuota := range quotas {
 			hardResources := quotav1.ResourceNames(resourceQuota.Hard)

